### ISSUE TITLE
Use statistics fot GPU(Radeon) doesnt show

### ISSUE THREAD

**Issue by APOS80** [openphotogrammetry/meshroomcl#12]
I have a Ryzen and Radeon setup.

The gpu i clearly used, as by looking in the radeon software work load visualicer, but noting is shown in meshroomcl under statistics:gpu.

**Comment by revisionarian** [comment]
You're right, the Meshroom GPU statistics are not recorded properly for non-NVIDIA GPUs. Meshroom uses the external program "nvidia-smi" to gather its GPU usage statistics, which does not work with non-CUDA GPUs. For MeshroomCL, we will need to find another way to gather the GPU stats - suggestions welcome!

For now, you can monitor MeshroomCL's live GPU usage with the Windows 10 Task Manager or a tool provided by your particular GPU vendor (like the Radeon software for AMD cards).
